spec/features/general_auto_grading_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe "Auto-grading", :type => :feature do
+
+  describe "Mission admin pages" do
+    let(:lecturer) {FactoryGirl.create(:lecturer)}
+    let!(:course) { FactoryGirl.create(:course, creator: lecturer) }
+    before do
+      sign_in lecturer
+    end
+
+    describe "Single question exact options" do
+      let(:mission) { FactoryGirl.create(:mission, :with_auto_graded_exact_general_questions, :completed, course: course) }
+      before do
+        visit edit_course_assessment_submission_grading_path(course, mission, mission.submission, mission.submission.gradings.first)
+        expect(page).to have_text('Grading: #{mission.title}')
+        expect(page).to have_selector('#auto-grading-exact')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.